### DISCUSSION TITLE
Conquérir l'écosysteme Docker sur Ubuntu : Installation expliquée

### DISCUSSION THREAD

**Discussion by giscus** [zero-filtre/zerofiltre-blog#200] [Announcements]
# Conquérir l'écosysteme Docker sur Ubuntu : Installation expliquée

Guide pratique pour pour découvrir le processus d'installation de docker ceci avec le systeme d'Exploitation Ubuntu pour développeurs de toute categorie.

https://zerofiltre.tech/articles/287

**Comment by imphilippesimo** [comment]
Plusieurs fautes de grammaire et d'orthographe à corriger.
Merci.

**Comment by imphilippesimo** [comment]
>C'est une plateforme open source populaire que l'on retrouve presque partout, comme sur les serveurs, les sites (Netflix, Target, Adobe..) etc...

Je ne comprends pas, 
Cette phrase n'est pas assez professionnelle, quand j'ouvre Netflix vois-je Docker ? Non
 Docker n'est-il que pour des env de test ? Non

Merci de revoir toute cette partie de définition de docker.  
Tu peux utiliser la définition qui se situe sur la documentation Docker

**Comment by imphilippesimo** [comment]
> Plus simplement, imagine que tu as une application à faire, et que ta machine n'a pas les capacités nécessaires pour la faire fonctionner. Alors, le « conteneur » pourra recueillir les dépendances nécessaires que ta machine n'a pas pour faire fonctionner ton application. 🫠 

1. pu faire fonctionner ...

2. Une machine n'ayant pas assez de ressources ne va pas mieux faire tourner une application tout simplement parce que c'est dans un container attention. Cependant tu pourras empaquetter ton appli dans un container pour la faire tourner ailleurs.

Merci de revoir cette partie et de faire relire l'article par quelqu'un d'autre afin que nous perdions pas de temps à valider n fois le même article.

**Reply by imphilippesimo** [reply]
Annule ma remarque 1, 2 et le reste valable.

**Comment by imphilippesimo** [comment]
> La machine virtuelle est semblable à un système d'exploitation complet,

C'est pas juste semblable, ça embarque un OS entier.

**Comment by imphilippesimo** [comment]
>Vous pouvez mieux comprendre les images Docker en les considérant comme des plans. Elles contiennent des instantanés de ce qu’un conteneur comprendra lorsqu’il s’exécutera.

Explique en termes de moule et gateau.

**Comment by imphilippesimo** [comment]
À la place d'Ubuntu parle des distributions Débian en général. En limitant à Ubuntu tu limites ton audience.

**Comment by imphilippesimo** [comment]
Enfin en quoi est ce que cet article est différent de la documentation docker ? 
Je le trouve léger, il n'apporte pas d'informations rares que l'on trouve difficilement ailleurs.

Nous tolérions ce type d'articles il y'a de celà 1 an mais il va falloir produire désormais des articles pointus avec des infos pas retrouveable n'importe où.

Nous pouvons vous fournir des sujets si vous le voulez en fonction de vos compétences.
Laissez-nous un mail pour en parler. 
Cet article sera publié après correction des fautes mais ce sera le dernier dans le genre.

Cordialement
